src/enums/eventType.ts

```diff
@@ -25,6 +25,7 @@ export enum EventType {
     Cipher_ClientAutofilled = 1114,
     Cipher_SoftDeleted = 1115,
     Cipher_Restored = 1116,
+    Cipher_ClientToggledHiddenFieldCount = 1117,
```

Comment:
Hi, please update/rebase latest and note that there's a new enum taking this value, you'll need to go up to the next. Also, this event type is an "audit" style event for logging and not something we would likely track (such as the same as showing a hidden field value) and if it automatically shows the hidden field value, it should simply re-use the existing event for that. Plus this would need to also exist via the API/server; so best to revert this change here.